app/assets/javascripts/admin/addon/routes/admin-route-map.js

```diff
@@ -97,6 +97,15 @@ export default function () {
             this.route("edit", { path: "/:field_name" });
           }
         );
+        this.route(
+          "adminCustomizeFormTemplates",
+          { path: "/form-templates", resetNamespace: true },
+          function () {
+            this.route("index", { path: "/" });
```

Comment:
This shouldn't be needed: https://guides.emberjs.com/release/routing/defining-your-routes/#toc_index-routes